docs: Add 2023-03-09 meeting notes

sam3k opened the pull request:
Closes #414

nzakas commented on notes/2023/2023-03-09.md:
```diff
@@ -0,0 +1,53 @@
+
+## Transcript
+
+
```
The transcript format is always the same, so you can put it in and trust that it will be uploaded eventually. :)

```suggestion
[`2023-03-09-transcript.md`](2023-03-09-transcript.md)
```

nzakas commented on notes/2023/2023-03-09.md:
```diff
@@ -0,0 +1,53 @@
+### Documentation-specific License
+**Summary:** We don't currently have a documentation-specific license, so it defaults to all rights reserved. 
+
+> @nkzakas:
```
```suggestion
> @nzakas:
```

nzakas commented on notes/2023/2023-03-09.md:
```diff
@@ -0,0 +1,53 @@
+> Agenda item: We don't currently have a documentation-specific license, so it defaults to all rights reserved. I'd like to propose we license the documentation as [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+
+
+**Resolution:** Okay, we've agreed to license the docs under Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. We will place a notice both in the docs README and in the footer of the docs site. @Sam3k can you please create an issue for that?
```
Let's call out when there's an action item and who is responsible for it.

```suggestion
**Resolution:** Okay, we've agreed to license the docs under Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. We will place a notice both in the docs README and in the footer of the docs site.

**Action Item:** @Sam3k to create an issue for this.
```

nzakas reviewed (dismissed):
This looks good! Just left a couple of suggestions.

Please be sure to follow up on creating the issue we mentioned, and on updating the RFC PR to remove the "tsc agenda" label and add the summary we discussed over Discord.

Thanks!

sam3k commented:
@nzakas:

- Accepted all your suggestions. Thank you!
- Just removed the `tsc-agenda` from [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- Left a TSC Summary comment in [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- Added `Final Commenting` label to [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- The issue has been [created](https://github.com/eslint/eslint/issues/16980) as well

Let me know if I've missed anything.

nzakas reviewed (approved):
LGTM. Just waiting for @mdjermanovic to approve before merging.

mdjermanovic reviewed (approved):
Thanks!
